bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-component-visibility",
   "main": "index.js",
-  "version": "0.0.5",
+  "version": "0.0.6",
```

Comment:
no need to pump this until a 0.0.6 tag actually exists on github, which I do when I release a new version, so let's leave this as 0.0.5 (also, bower technically ignoes the version *entirely*)